micah-akpan/questioner-api
# #162952771 Admin can add Tags to Meetup Feature

#### What does this PR do?
Have the admin add tag feature working as expected
#### Description of Task to be completed?
- Admins can add tags to a meetup
#### How should this be manually tested?
- clone repo
- Add database env variables listed in the ReadME (will be implemented soon)
- ```npm install```
- ```npm run dev:start```
- Using Postman, go to POST /api/v2/meetups with the right auth headers
- Provide the required fields, and a tag field, this should add tags to the newly meetup
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#162952771
#### Screenshots (if appropriate)
#### Questions:
